.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+cache: pip
+
+matrix:
+  include:
+    - env: TOXENV=py27-dj18
+      python: 2.7
+    - env: TOXENV=py27-dj110
+      python: 2.7
+    - env: TOXENV=py35-dj18
```

Comment:
I wonder what the right answer is for testing against 3.x Python's. My machine doesn't have 3.5, for example (i have 3.6 via Homebrew)